README.md

```diff
@@ -30,6 +30,50 @@ latest JavaScript features; please also see [this wiki page](https://github.com/
 
 + Older browsers: https://mozilla.github.io/pdf.js/legacy/web/viewer.html
 
+## Setup Instructions (Platform-Specific)
+
+### Windows
+1. Install [Node.js](https://nodejs.org/) and [npm](https://www.npmjs.com/get-npm).
+2. Clone the repository and navigate into the project folder:
+   bash
+   git clone https://github.com/mozilla/pdf.js
+   cd pdf.js
+3. Install dependencies
+   npm install
+   npm start
```

Comment:
Also, this seems wrong/unnecessary since our https://github.com/mozilla/pdf.js/blob/master/package.json file doesn't not have a section that would necessitate this command; please see https://docs.npmjs.com/cli/v10/commands/npm-start